starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -809,16 +810,13 @@ sub large_dither_checks {
     my $self = shift;
     my $dither_state = shift;
     my $all_dither = shift;
-    my $time_tol = 10;         # Commands must be within $time_tol of expectation
+    my $time_tol = 11;         # Commands must be within $time_tol of expectation
 
     # Save the number of warnings when starting this method
     my $n_warn = scalar(@{$self->{warn}});
 
-    # The obs tstart and tstop times are derived from the OFLS maneuver summary file,
-    # which for implementation reasons gives the maneuver times offset by 10 seconds
-    # from the actual command timing. Fix this here.
-    my $obs_tstart = $self->{obs_tstart} - 10;
```

Comment:
It looks like the maneuver summary parsing is adding 10 seconds to tstart but not tstop?

https://github.com/sot/starcheck/blob/08cf90fbb57534c126bc5111fa0b36ab7c5deb26/starcheck/src/lib/Ska/Parse_CM_File.pm#L724

Either way, subtracting it off this way was not working on this test against big dither commanding in load explicitly verified by PCAD.